### PR TITLE
Added delete chores method and unit tests for delete chores.

### PR DESCRIPTION
**Pull request by Ahsen19**

### CODE REVIEW

File: server/controller/chore.js [hunk 1]
```diff
@@ -157,3 +158,39 @@ export const editChore = async (req, res) => {
...
+      // assigning the updated array to user.chores
+      user.chores = userChores; 
+      await user.save();
+      return res.status(200).json({ message: `Chore with id ${choreId} deleted successfully!`});
+    }
+    else {
+      console.log("Chore ID not found in user's chore list!")
+      return res.status(500).json({ message: "Internal Server Error" });
+    }
+  }catch (error) {
```
Comment (JoshuaDueck) [inline server/controller/chore.js:191]: Indentation and spacing is inconsistent here (else, and the try-catch). Did you run prettier? I'd look into getting that set up. I have my VSCode set to format on save.

**Review by JoshuaDueck — APPROVED** [review]
Aside from the comments, looks good to me! I approve when the comments are resolved.

**Comment by JoshuaDueck** [comment]
Tests are failing. We should look into that.

File: server/controller/chore.js [hunk 2]
```diff
@@ -157,3 +158,39 @@ export const editChore = async (req, res) => {
...
+ ** @param {Object} req - The request object
+ ** @param {Object} res - The response object
+ */
+export const deleteChore = async (req, res) => {
+  try{
+    const choreId = req.params.id;
+    // delete chore from chores database
+    await Chore.findByIdAndDelete({ _id: choreId });
+
+    // get the user that is creating the chore
```
[the remark this answers is not in the document]
Comment (Ahsen19) [reply server/controller/chore.js:173]: Hahaha good catch ill fix it

File: server/controller/chore.js (hunk 1 above)
Comment (Ahsen19) [reply server/controller/chore.js:191]: No i didnt but thats a good point

**Comment by Ahsen19** [comment]
Only one test I wrote is failing. Will fix that later but merging it now so it dosnt fall to far behind

**Pull request merged**